src/api/grainConfig.test.js

```diff
@@ -94,6 +95,38 @@ describe("api/grainConfig", () => {
       expect(() => toDistributionPolicy(x)).not.toThrow();
     });
 
+    it("default immediate's numPeriodsLookback to 1", () => {
```

Comment:
nit: this test would be cleaner to set everything except `immediatePerWeek` to 0, so there'd be less noise in the `expectedDistributionPolicy`.

(no change required)

Reply:
Done, this makes a lot of sense.